Package.swift

```diff
@@ -42,6 +42,9 @@ let package = Package(
         .product(name: "OrderedCollections", package: "swift-collections"),
         .product(name: "SwiftUINavigationCore", package: "swiftui-navigation"),
         .product(name: "XCTestDynamicOverlay", package: "xctest-dynamic-overlay"),
+      ],
+      resources: [
+        .process("PrivacyInfo.xcprivacy")
```

Comment:
Does this need to be added to [Package@swift-5.9.swift](https://github.com/pointfreeco/swift-composable-architecture/blob/main/Package%40swift-5.9.swift), too?

Reply:
I believe so?

For any TCA users publishing an app using 5.9 after the App Store rules change. 

(Sorry, I don't have experience with @ files for Swift Packages, but I assume it's a Package.swift file used for people still developing in a Swift 5.9 environment).

Reply:
@DandyLyons Can you add those lines there, as well? Basically anyone on a modern Xcode will be using the -5.9.swift version (the other version you updated will only be used by <5.9).

Reply:
I pushed it quick to your branch, will merge assuming CI looks generally good! I think the shared state branch CI isn't totally green right now, so no worries if it partially fails.

Reply:
Thanks. Sorry. I was out of the house. Let me know if I can help with anything else.